ChromecastSampleApp/ChromecastSampleApp/OOCastManagerFetcher.m

```diff
@@ -11,6 +11,6 @@
 
 @implementation OOCastManagerFetcher
 +(OOCastManager*) fetchCastManager {
-  return [OOCastManager getCastManagerWithAppID:@"D4C7BD4A" namespace:@"urn:x-cast:ooyala"];
+  return [OOCastManager getCastManagerWithAppID:@"B3560E04" namespace:@"urn:x-cast:ooyala"];
```

Comment:
It was staging custom receiver, you replaced it by product default receiver.  Shouldn't it be product custom receiver 4172C76F ?